plugins/com.google.cloud.tools.eclipse.appengine.login.test/src/com/google/cloud/tools/eclipse/appengine/login/ui/AccountSelectorTest.java

```diff
@@ -208,6 +207,93 @@ public void testSelectAccount_nullOrEmptyEmail() {
   }
 
   @Test
+  public void testSelectAccountInSingleAccountSelectMode_multipleAccounts() {
+    when(loginService.getAccounts())
+        .thenReturn(new HashSet<>(Arrays.asList(account1, account2, account3)));
+    AccountSelector selector =
+        new AccountSelector(shell, loginService, "<select this to login>", true);
+
+    selector.selectAccount("some-email-2@example.com");
+    assertNotEquals(-1, selector.combo.getSelectionIndex());
```

Comment:
The combo isn't being kept in any kind of order.  I'll make it sort alphabetically.